spec/helpers/claims_helper_spec.rb

```diff
@@ -202,4 +202,28 @@ def current_user
       it { expect(headings.keys).not_to include(:page_notice) }
     end
   end
+
+  describe '#misc_fees_locals' do
+    subject(:locals) { misc_fees_locals(claim) }
+
+    let(:claim) { build :claim }
+
+    context 'with a claim eligible for unused materials fees' do
+      before { allow(claim).to receive(:unused_materials_applicable?).and_return true }
+
+      it { expect(locals[:unclaimed_fees_notice]).not_to be_nil }
```

Comment:
prefer positive tests if possible plus you could test the value which would have the benefit of testing the translation exists and has the expected content.
```
it { expect(locals.keys).to include(:unclaimed_fees_notice) }
it { expect(locals[:unclaimed_fees_notice]).to eql('blah'))
```